src/platforms/php/guides/symfony/index.mdx

```diff
@@ -17,32 +17,106 @@ Install the `sentry/sentry-symfony` package:
 
-Add your DSN to `app/config/config.yml`:
+```yaml
+sensio_framework_extra:
+   psr_message:
+      enabled: false
+```
+
+For more details about the issue see [https://github.com/sensiolabs/SensioFrameworkExtraBundle/pull/710](https://github.com/sensiolabs/SensioFrameworkExtraBundle/pull/710).
+</Alert>
+
+## Configure
+
+Add your DSN to `config/packages/sentry.yaml`:
 
-```yaml {filename:app/config/config.yml}
+```yaml {filename:config/packages/sentry.yaml}
 sentry:
   dsn: "___PUBLIC_DSN___"
 ```
 
-If you **are not** using Symfony Flex, you'll also need to enable the bundle in `app/AppKernel.php`:
+If you **are not** using Symfony Flex, you'll also need to enable the bundle in `config/bundles.php`:
+
+```php {filename:config/bundles.php}
+<?php
+
+return [
+    // ...
+    Sentry\SentryBundle\SentryBundle::class => ['all' => true],
+];
+```
+
+### Monolog Integration
+
+If you are using monolog, these additional configuration and services are needed to log the errors correctly:
+
+```yaml {filename:config/packages/sentry.yaml}
+sentry:
+    register_error_listener: false # Disables the ErrorListener to avoid duplicated log in sentry
+    dsn: "%env(SENTRY_DSN)%"
+
+monolog:
+    handlers:
+        sentry:
+            type: service
+            id: Sentry\Monolog\Handler
+
+services:
+    Sentry\Monolog\Handler:
+        arguments:
+            $hub: '@Sentry\State\HubInterface'
+            $level: !php/const Monolog\Logger::ERROR
+```
+
+Additionally, you can register the `PsrLogMessageProcessor` to resolve PSR-3 placeholders in reported messages:
+
+```yaml {filename:config/packages/sentry.yaml}
+services:
+    Monolog\Processor\PsrLogMessageProcessor:
+        tags: { name: monolog.processor, handler: sentry }
+```
+
+## Test the implementation
+
+To test that both logger error and exception are correctly send to sentry
+we can create the following controller:
 
-```php {filename:app/AppKernel.php}
+```php
 <?php
-class AppKernel extends Kernel
+
+namespace App\Controller;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+class SentryTestController
 {
-    public function registerBundles()
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    public function __construct(LoggerInterface $logger)
     {
-        $bundles = array(
-            // ...
+        $this->logger = $logger;
+    }
 
-            new Sentry\SentryBundle\SentryBundle(),
-        );
+    /**
+     * @Route(name="sentry_test", path="/_sentry-test")
+     */
+    public function testLog(Request $request)
+    {
+        $this->logger->error('My custom logged error.');
 
-        // ...
+        throw new \RuntimeException('Example exception.');
     }
-
-    // ...
 }
 ```
+
+After you visit now `/_sentry-test` this **both** errors should appear on sentry.
```

Comment:
```suggestion
After you visit the `/_sentry-test` page, you should see two errors logged in Sentry.
```

However, this example is a bit tricky because the first error will be logged only if you are using the Monolog handler. Either we should specify this with a little comment in the code, or we should change the example itself to just use the `throw new` as it's enough to work in both cases